src/client/lib/utils.js

```diff
@@ -25,7 +25,7 @@ export var deserialize = (str) => JSON.parse(str || '{}');
  * @params {Object} obj: The Object to be cloned.
  * @return {Object} Deep clone of the input Object.
  */
-export var clone = (obj) => deserialize(serialize(obj));
+export let clone = (obj) => deserialize(serialize(obj));
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).